### PR TITLE
Api updates

### PR DESCRIPTION
**Pull request by eliascodes**
Related to #447 #486 #484

#### Changes
* Fix API calls on QR code page (visitor "sign in")
* Update `Visitors.update` call on `VisitorProfile` page.
* Update docs

##### Updating Existing Features
* [ ] Have I manually tested my changes?
* [x] Have I added/removed/updated the test suite for this feature appropriately?
* [x] Have I updated the project documentation and README to reflect my changes?
* [x] Do my changes affect steps towards the desired project architecture?

### CODE REVIEW

File: client/src/api/index.js [hunk 1]
```diff
@@ -57,19 +57,16 @@ export const Visitors = {
...
       {
-        userId: id,
-        userFullName: name,
-        sex: gender,
-        yearOfBirth: yob,
+        name,
+        gender,
         email,
+        phoneNumber,
+        birthYear: yob,
```
Comment (astroash) [inline client/src/api/index.js]: why not change yob -> birthYear for consistency?

File: client/src/visitors/pages/qrcode.js [hunk 2]
```diff
@@ -136,9 +138,18 @@ export default class QRCode extends Component {
...
+    }
+
     Visitors.createVisit({
+      activityId: activity.id,
+      visitorId: this.state.visitorId,
       qrCode: this.state.qrCodeContent,
-      activity: newActivity,
     })
       .then(() => this.props.history.push('/visitor/end'))
       .catch((error) => {
```
Comment (astroash) [inline client/src/visitors/pages/qrcode.js]: 👇  can this console.log error message be updated?

File: client/src/visitors/pages/qrcode.js [hunk 3]
```diff
@@ -136,9 +138,18 @@ export default class QRCode extends Component {
   changeActivity = (newActivity) => {
     this.setState({ activity: newActivity });
```
Comment (astroash) [inline client/src/visitors/pages/qrcode.js]: If I'm reading this correctly... The component that fires this function as an onclick is grabbing the activity value from state, which is being set in this function...

File: client/src/visitors/pages/qrcode.js [hunk 4]
```diff
@@ -175,15 +186,15 @@ export default class QRCode extends Component {
...
         <StyledSection margin={3}>
           <BigFlexContainerRow>
             {this.state.activities
               .map((activity, index) => (
                 <PurposeButton
-                  key={activity.name}
+                  key={activity.id}
                   color={index}
                   session={activity.name}
                   activity={this.state.activity}
```
Comment (astroash) [inline client/src/visitors/pages/qrcode.js]: 🙈

File: client/src/visitors/pages/qrcode.js (hunk 3 above)
Comment (eliascodes) [reply client/src/visitors/pages/qrcode.js]: Lol, yeah, don't ask me, I didn't write it 🤷🏼‍♂️

**Pull request merged**